src/libraries/permissionUtils/usePermission.ts

```diff
@@ -7,6 +7,8 @@ import { usePermissions } from "./usePermissions";
  */
 export const usePermission = (name: TPermission): boolean => {
   const permissions = usePermissions();
-
+  if (!process.env.ENABLE_PERMISSIONS) {
```

Comment:
Please declare this variable also in the .env file, in order to expose this option. I would prefer permissions to e enabled by default.